third_party/Makefile

```diff
@@ -29,3 +29,8 @@ bandwidthTest: sb_micro_path
 	git clone -b v$(shell nvcc --version | grep 'release' | awk '{print $$6}' | cut -c2- | cut -d '.' -f1-2) https://github.com/NVIDIA/cuda-samples.git ./cuda-samples
 	cd ./cuda-samples/Samples/bandwidthTest && make clean && make TARGET_ARCH=x86_64 SMS="70 75 80 86"
 	cp -v ./cuda-samples/Samples/bandwidthTest/bandwidthTest $(SB_MICRO_PATH)/bin/
+# Build FIO (3.27).
+fio: sb_micro_path
+ifneq (,$(wildcard fio/Makefile))
+	cd ./fio && ./configure --prefix=$(SB_MICRO_PATH) && make -j8 && make install
```

Comment:
hard code `8` makes it slow when running on 96 cores

Reply:
fixed